eth-contracts/contracts/DelegateManager.sol

```diff
@@ -361,9 +362,10 @@ contract DelegateManager is InitializableV2 {
 
         // Total valid funds used to calculate rewards distribution
         uint totalActiveFunds = (
-            totalBalanceOutsideStaking.sub(spDelegateInfo[msg.sender].totalLockedUpStake)
+            totalBalanceOutsideStaking.sub(
+                spDelegateInfo[msg.sender].totalLockedUpStake.add(spLockedStake)
```

Comment:
can you just use `totalRewards` here instead of recalculating?

Reply:
definitely not - this represents all locked up stake, which is unrelated to totalRewards

Reply:
i meant `totalLockedUpState`

Reply:
return totalLockedUpStake and calculate with that value instead of re-computing

Reply:
switched to return totalActiveFunds eliminating this calculation entirely